packages/backend-split/src/experimental/instanceMetadata.ts

```diff
@@ -18,22 +18,32 @@ import {
   createBackendPlugin,
 } from '@backstage/backend-plugin-api';
 import { instanceMetadataServiceRef } from '@backstage/backend-plugin-api/alpha';
+import Router from 'express-promise-router';
 
-// Example usage of the instance metadata service to log the installed features.
+// Example usage of the instance metadata service to log the installed features and host a small HTTP API.
 export default createBackendPlugin({
-  pluginId: 'instance-metadata-logging',
+  pluginId: 'instance-metadata',
   register(env) {
     env.registerInit({
       deps: {
         instanceMetadata: instanceMetadataServiceRef,
         logger: coreServices.logger,
+        httpRouter: coreServices.rootHttpRouter,
       },
-      async init({ instanceMetadata, logger }) {
+      async init({ instanceMetadata, logger, httpRouter }) {
         logger.info(
           `Installed features on this instance: ${instanceMetadata
             .getInstalledFeatures()
             .join(', ')}`,
         );
+
+        const router = Router();
+
+        router.get('/features/installed', (_, res) => {
+          res.json({ items: instanceMetadata.getInstalledFeatures() });
+        });
+
+        httpRouter.use('/.backstage/instanceInfo', router);
```

Comment:
```suggestion
        httpRouter.use('/.backstage/instanceMetadata/v1', router);
```

i think we started doing versioning for services. and probably makes sense to name like the service itself.